infoschema/perfschema/const.go

```diff
@@ -454,6 +455,71 @@ const tableEventsStatementsSummaryByDigest = "CREATE TABLE if not exists events_
 	"LAST_SEEN TIMESTAMP(6) NOT NULL," +
 	"QUERY_SAMPLE_TEXT LONGTEXT DEFAULT NULL);"
 
+// tableEventsStatementsSummaryByDigestHistory contains the column name definitions for table
+// events_statements_summary_by_digest_history.
+const tableEventsStatementsSummaryByDigestHistory = "CREATE TABLE if not exists events_statements_summary_by_digest_history (" +
```

Comment:
the schema of `events_statements_summary_by_digest_history ` is same with `events_statements_summary_by_digest`? 

A little redundant here, but i have no idea to solve this....

Reply:
Yes, it's the same. Would it be better to write `const tableEventsStatementsSummaryByDigestHistory = tableEventsStatementsSummaryByDigest`?

Reply:
But you need to change the table name ...

Reply:
I can use `strings.Replace`, but I don't think it's necessary.